docs/codeql/reusables/download-github-database.rst

```diff
@@ -7,7 +7,7 @@ For example, to check for CodeQL databases using the `GitHub CLI <https://cli.gi
 
 This command returns information about any CodeQL databases that are available for a repository, including the language the database represents, and when the database was last updated. If no CodeQL databases are available, the response is empty.
 
-When you have confirmed that a CodeQL database exists for the language you are interested in, you can download it using the following command:: 
+When you have confirmed that a CodeQL database exists for the language you are interested in, you can download it using the following command::
 
    gh api /repos/<owner>/<repo>/code-scanning/codeql/databases/<language> -H 'Accept: application/zip' > path/to/local/database.zip
```

Comment:
That link should be correct. It did appear on the docs page a few days ago. I've asked the docs team for assistance.
Since the problem is on docs.github.com we can merge this still merge this.